addons/binding/org.openhab.binding.nibeheatpump/ESH-INF/thing/f1x45.xml

```diff
@@ -54,6 +54,11 @@
 				<description>Comma separated list of registers, which are allowed to write to heat pump. E.g. 44266, 47004</description>
 				<default></default>
 			</parameter>
+            <parameter name="throttleTime" type="integer">
```

Comment:
You can add a `unit` attribute here to make clear which value is expected (see [configuration descriptions - `parameter.unit`](https://www.eclipse.org/smarthome/documentation/development/bindings/xml-reference.html#xml-structure-for-configuration-descriptions).

Reply:
Done